AndelaOSP/art-backend
# [#160983018] Upgrade Django to fix CVE-2018-6188 security vulnerability

### What does PR do?

- Upgrades Django to fix CVE-2018-6188 security vulnerability

### Description of the task to be completed:
  Upgrade django to the latest release.
 ### How can this be manually tested?
  Run `python manage.py shell` then Import `django`. 
  executing `django.__version__` should print `2.1.2`
  Also run all tests to ensure they are all passing.
### Relevant pivotal tracker stories
[#160983018](https://www.pivotaltracker.com/story/show/#160983018)
